src/nbl/video/COpenGL_LogicalDevice.h

```diff
@@ -135,7 +135,33 @@ class COpenGL_LogicalDevice : public IOpenGL_LogicalDevice
 
         return retval;
     }
+    core::smart_refctd_ptr<IGPUImage> createImage(IGPUImage::SCreationParams&& params) override final
+    {
+        if (!asset::IImage::validateCreationParameters(params))
+            return nullptr;
+        if constexpr (IsGLES)
+        {
+            if (params.type == IGPUImage::ET_1D)
+                return nullptr;
+        }
+
+        core::smart_refctd_ptr<IGPUImage> retval;
+
+        SRequestImageCreate2 reqParams;
 
+        reqParams.mreqs.size = 0u; // TODO(Erfan) some approx of image size -> considering dimensions, mipLevels, arrayLayers, samples, minImageGranularity and texelBlockInfo (see ImageUploadUtility)
+        reqParams.mreqs.memoryTypeBits = m_physicalDevice->getDeviceLocalMemoryTypeBits();
+        reqParams.mreqs.alignmentLog2 = 0u; // TODO(Erfan) Alignment previously was 0u in getXXXMemoryRequirementsOnDedMem(). what to set here? get minXXXOffsetAlignment from physical device and deduce from usage?
+        reqParams.mreqs.prefersDedicatedAllocation = true;
+        reqParams.mreqs.requiresDedicatedAllocation = true;
+
+        reqParams.creationParams = std::move(params);
+        auto& req = m_threadHandler.request(std::move(reqParams), &retval);
+        m_masterContextCallsInvoked++;
+        m_threadHandler.template waitForRequestCompletion<SRequestImageCreate>(req);
```

Comment:
typo, should be 
```cpp
m_threadHandler.template waitForRequestCompletion<SRequestImageCreate2>(req);
```

request type sent must match the template param

Reply:
tbf safest way to code it is
```cpp
m_threadHandler.template waitForRequestCompletion<decltype(reqParams)>(req);
```
P.S. I might be missing an `std::remove_reference_t<decltype(reqParams)>`